lib/web_ui/lib/src/engine/engine_canvas.dart

```diff
@@ -33,7 +33,7 @@ abstract class EngineCanvas {
 
   void transform(Float32List matrix4);
 
-  void clipRect(ui.Rect rect);
+  void clipRect(ui.Rect rect, ui.ClipOp clipOp);
```

Comment:
We can add comment here since it's a public method, or we can add the comments to the implementing classes.

Reply:
At some point all comments were removed so we didn't have to keep them in sync.